### ISSUE TITLE
Use the new Mux Player

### ISSUE THREAD

**Issue by stipsan** [sanity-io/sanity-mux-player#25]
Mux have released a [new player](https://www.mux.com/player).

It implements Mux Data and is overall a really good player.

Relates to:
- #23
- #19
- #21
- #10

**Comment by dylanjha** [comment]
@stipsan 🙌🏼

Do you think it might make sense to deprecate the sanity-mux-player and simply point folks directly to Mux Player?

If sanity-mux-player is just a wrapper around Mux Player then it's not really doing a whole lot and it would be one more package to maintain and keep up to date every time there is a new Mux Player release. Which is ~every week right now. 

Just a thought. Thanks for opening the issue, I was wanting to follow up with y'all about this 🙌🏼

**Comment by stipsan** [comment]
@dylanjha that would be brilliant ☺️ does the mux player give feedback on statuses such as if it's in `preparing` right after it's been uploaded?

**Comment by dylanjha** [comment]
Yep, Mux Player will catch common error states, log it to the dev console and link to more detailed error page. 

If an asset is still preparing it will link to this 412 error: https://github.com/muxinc/elements/blob/main/errors/412-not-playable.md

(These common error states also show up in Mux Data)
